lighthouse-core/index.js

```diff
@@ -37,8 +37,8 @@ async function lighthouse(url, flags, configJSON) {
   flags = flags || /** @type {LH.Flags} */ ({});
   i18n.setLocale(flags.locale);
 
-  // set logging preferences, assume quiet
-  flags.logLevel = flags.logLevel || 'error';
+  // set logging preferences, assume the user is handling it
+  flags.logLevel = flags.logLevel || 'custom';
```

Comment:
feels like we shouldn't change this now? It's possible someone depends on this. We can make it available for those who want it, though

Reply:
This really is the most sane default for node module consumption though (which is the only channel it should affect). Just a TODO for next major then? 😢

Reply:
You might be right, if you're using the node module and you care about the logging, you're probably most likely to have set `quiet`, but it just might be surprising why errors are suddenly not logged anymore?

Reply:
these modes are awful if you're actually using debug though because it disables all other modules output. I suppose we'll just add custom to the notes of the issue